packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -43,8 +43,6 @@ query TrackedBanner_ProjectState {
 gql`
 mutation TrackedBanner_SetProjectState($value: String!) {
   setPreferences(type: project, value: $value) {
-    ...TestingPreferences
-    ...SpecRunner_Preferences
```

Comment:
Why removed?

Reply:
ah, I removed the wrong ones thanks for catching this